modules/portworx-data-services/scripts/rm-pds-entry.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -u
+set -o pipefail
+. vars
```

Comment:
there's no vars file on the terraform script and therefore below kbCtl and vKubeConfig aren't set